apiUtility/apiUtils.py

```diff
@@ -17,8 +17,10 @@
 from scripts.message import store_sms_data, store_sms_messages
```

Comment:
Add relevant comments at the top and above each function as discussed earlier

Reply:
Comments added to location.py

Reply:
I will comment on other files separately sir!